include/no_os_i2c.h

```diff
@@ -74,11 +91,33 @@ struct no_os_i2c_init_param {
 	void		*extra;
 };
 
+/**
+ * @struct no_os_i2cbus_desc
+ * @brief Structure holding I2C bus descriptor
+ */
+struct no_os_i2cbus_desc {
+	/** I2C bus mutex(lock)*/
+	void* mutex;
+	/** I2C bus Device ID */
+	uint32_t	device_id;
+	/** I2C bus maximum transfer speed supported */
+	uint32_t	max_speed_hz;
+	/** I2C bus Slave address */
+	uint8_t		slave_address;
+	/** I2C bus platform specific functions */
+	const struct no_os_i2c_platform_ops *platform_ops;
+	/** I2C bus extra parameters (device specific parameters) */
+	void		*extra;
+};
+
+
 /**
  * @struct no_os_i2c_desc
- * @brief Structure holding I2C descriptor
+ * @brief Structure holding I2C address descriptor
```

Comment:
is this change required?

Reply:
Do you have a better naming version? It should differentiate the bus descriptor from the i2c descriptor that has address associated to it